hepcrawl/parsers/jats.py

```diff
@@ -464,6 +466,7 @@ def get_root_node(jats_record):
         else:
             root = jats_record
         root.remove_namespaces()
+        etree.cleanup_namespaces(root.root)
```

Comment:
you shouldn’t readd this, it’s not necessary anymore as it has been included in parsel.

Reply:
okok, I got it from another PR where it's being added. I'll remove.